doc/examples/edges/plot_active_contours.py

```diff
@@ -37,12 +37,13 @@
 img = rgb2gray(img)
 
 s = np.linspace(0, 2*np.pi, 400)
-x = 220 + 100*np.cos(s)
-y = 100 + 100*np.sin(s)
-init = np.array([x, y]).T
+r = 100 + 100*np.sin(s)
```

Comment:
Spaces around *

Reply:
@ahojnnes I want to minimise how much I mess with existing code, and anyway PEP8 explicitly allows not having the spaces to indicate operator priority.

Reply:
Thanks for the review though, good to see you around! ;)